Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -901,8 +901,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="Debugger\RemoteDebug\RemoteDebug.js">
+    <Content Include="RemoteDebug\RemoteDebug.js">
```

Comment:
The msi installs the files to just `RemoteDebug`. Should I change the msi instead?